### PR TITLE
[bitnami/git] Add VIB tests to Git container

### PR DESCRIPTION
**Pull request by mdhont**
### Description of the change

Adding the VIB tests to the Git container

### CODE REVIEW

File: .vib/git/goss/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../git/goss/git.yaml: {}
+  # Load scripts from .vib/common/goss/templates
+  ../../common/goss/templates/check-app-version.yaml: {}
+  ../../common/goss/templates/check-binaries.yaml: {}
+  ../../common/goss/templates/check-broken-symlinks.yaml: {}
+  ../../common/goss/templates/check-ca-certs.yaml: {}
+  ../../common/goss/templates/check-linked-libraries.yaml: {}
+  ../../common/goss/templates/check-sed-in-place.yaml: {}
```
Comment (FraPazGal) [inline .vib/git/goss/goss.yaml:10]: Please add `check-spdx.yaml`

File: .vib/git/goss/git.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+command:
```
Comment (FraPazGal) [inline .vib/git/goss/git.yaml:1]: These tests don't seem to be retriable.

Comment (mdhont) [reply .vib/git/goss/git.yaml:1]: I've added a random folder

Comment (FraPazGal) [reply .vib/git/goss/git.yaml:1]: Have you tested this? IIRC, using `$RANDOM` was not possible in Goss (note it is a bash function, not and env). Please review this; the alternative we have always used here is the go template function `randAlpha`.

File: bitnami/git/2/debian-11/Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@ ENV OS_ARCH="${TARGETARCH:-amd64}" \
 
 COPY prebuildfs /
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-# Install required system packages and dependencies
+## Install required system packages and dependencies
```
Comment (joancafom) [inline bitnami/git/2/debian-11/Dockerfile:20]: Please remove this change

**Review by joancafom — APPROVED** [review]

**Pull request merged**